boards/arm/mimxrt1050_evk/doc/mimxrt1050_evk.rst

```diff
@@ -23,7 +26,7 @@ basic 2D graphics, camera interface, SPDIF, and I2S audio interface.
 Hardware
 ********
 
-- MIMXRT1052DVL6A MCU (600 MHz, 512 KB TCM)
+- MIMXRT1052DVL6B MCU (600 MHz, 512 KB TCM)
```

Comment:
We still need to add the A1 silicon part numbers to https://github.com/zephyrproject-rtos/zephyr/blob/55b462cdfa8dcda620dc1cc984137ee1887e92a7/soc/arm/nxp_imx/rt/Kconfig.soc#L54-L59

But that depends on the next MCUXpresso SDK release.

Until then, let's keep this document to be about A0 silicon and EVK board, but make a note that the A0/EVK Zephyr configuration also works for A1/EVKB.

Reply:
agree, @MaureenHelm , the part numbers will be updated after the next SDK mainline release.